docs/RUNNING-LOCALLY.md

```diff
@@ -20,7 +20,7 @@ Prerequisites
 - [Rimraf](https://github.com/isaacs/rimraf)
 - [Asar](https://github.com/electron/asar)
 - [NSIS v2.51](http://nsis.sourceforge.net/Main_Page) (v3.x won't work)
-- [Visual Studio Community 2013](https://www.visualstudio.com/en-us/news/vs2013-community-vs.aspx)
+- [Visual Studio Community 2015](https://go.microsoft.com/fwlink/?LinkId=691978&clcid=0x409) (free) (other editions, like Professional and Enterprise, should work too)
```

Comment:
Ergh, that's a direct-download link (clicking it causes my browser to automatically download a .exe file). Do not like.
Perhaps the link above could be:
```[Visual Studio 2015](https://www.visualstudio.com/downloads/) (the Community edition is free)```